### ISSUE TITLE
Add `--version` flag for root command

### ISSUE THREAD

**Issue by jcbhmr** [fontist/fontist#354]
There's currently no `--version` command. This is a common flag for tools that are intended to be installed globally so that users can run `<command> --version` instead of remembering that they installed it with $X package manager and digging into the package manager to determine the version.

```sh
jcbhmr@PIG-2016:~$ fontist --version
Could not find command "__version".
```

**Comment by ronaldtse** [comment]
Agreed. `fontist version` and `fontist --version` should return the actual version of Fontist code. Should we also return the Formula's repo commit/date to indicate when it was last updated?

**Comment by alexeymorozov** [comment]
I agree about the repo commit/date. Also it may print [the current formulas branch](https://github.com/fontist/fontist/blob/v1.20.0/lib/fontist.rb#L52), it may be useful in [formulas CI](https://github.com/fontist/formulas/blob/e7464778e45683f4e643284974dba11d4267e1b6/.github/workflows/metanorma.yml#L37).
